### PR TITLE
Added testsuite docs

### PR DESCRIPTION
**Pull request by vitalych**
Signed-off-by: Vitaly Chipounov <vitaly@cyberhaven.io>

### CODE REVIEW

File: src/Testsuite.rst [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+        s2e testsuite run
+
+   This command automatically starts the optimal number of parallel tests depending on how many
+   CPU cores and memory the machine has.
+
+   You may also specify one or more test project names, in which case the command will
+   run only those tests.
+
+   This command saves the console output of all tests in ``stdout.txt`` and ``stderr.txt`` files in the test project's
```
Comment (adrianherrera) [inline src/Testsuite.rst]: Can we please stick to a 120 char limit

Comment (vitalych) [reply src/Testsuite.rst]: Yes. This could be a jenkins check.

Comment (vitalych) [reply src/Testsuite.rst]: Actually it was fine, maybe it's github that truncates the line.

Comment (adrianherrera) [reply src/Testsuite.rst]: Oh ok. Yeah it looks weird in github

**Pull request merged**